src/plugins/internalnotification/README.md

```diff
@@ -0,0 +1,92 @@
+- infos = Plugin for internal notification
+- infos/author = Thomas Wahringer <waht@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = postgetstorage postcommit
+- infos/status = global libc unittest concept unfinished experimental nodoc
+- infos/metadata =
+- infos/description = Plugin for internal notification
+
+## Usage ##
+
+Allows applications to automatically update registered variables when the value of a specified key has changed.
+Support for callbacks that fire when a specified key has changed is planned.
```

Comment:
Move this to TODOs/Limitations. Please give some big picture in the beginning and then describe things that work.